helm/korifi/values.yaml

```diff
@@ -9,6 +9,8 @@ containerRegistrySecrets:
 eksContainerRegistryRoleARN: ""
 containerRegistryCACertSecret:
 systemImagePullSecrets: []
+tolerations: []
```

Comment:
The new values configure the cluster itself, I wonder whether it would make sense fo group them under a `k8s` section. I.e. instead of using `--set nodeSelector.role=something`, use `--set k8s.nodeSelector.role=something`

Korifi itself does not have the concept of tolerations and node selectors, so having those in the values "root" seems a bit odd to me.